buidl/tx.py

```diff
@@ -88,6 +88,16 @@ def dump_cache(cls, filename):
             s = json.dumps(to_dump, sort_keys=True, indent=4)
             f.write(s)
 
+            
+class TxSender:
+    @classmethod
+    def sendrawtransaction(cls, signed_tx_hex_str, network="mainnet"):
```

Comment:
Why is this a `@classmethod`? It doesn't use any instance of the `cls` below. It's dependent on `TxFetcher`, so should probably just be a method of that class. That said, it would kind of change `TxFetcher` to something like `TxFerryer` which is a mouthful. Maybe it would make the most sense to just keep the name as `TxFetcher` for backwards compatibility.

Reply:
Thanks for considering the PR.

Moving `sendrawtransaction()` to `TxFetcher` and keeping the name `TxFetcher` for backwards compatibility makes sense to me as well.  I'll add the change.

A few options were considered (creating `sendrawtransaction()` outside of a class, placing it under `TxFetcher` with nonideal naming, or placing it into a new but similar class `TxSender`).  The `@classmethod` usage was simply intended to align with the style of TxFetcher (e.g., `get_url()` is similar in that it doesn't appear to use `cls`).  Using `@staticmethod` seems like a better fit, since currently it is not anticipated that `sendrawtransaction()` would need access to class state.